src/web3/ternoaConnect.tsx

```diff
@@ -133,7 +133,7 @@ export default function TernoaConnect() {
                 // Here we will await the Wallet's response to the pairing proposal
                 const session = await approval();
                 onSessionConnected(session);
-                const nonce = await generateNonce(localStorage.getItem("walletTernoa"));
+                const nonce = await generateNonce(localStorage.getItem("walletTernoa") || '');
```

Comment:
Nous sommes après l'approbation de la connexion avec WalletConnect, on doit toujours connaitre l'adresse à cette endroit

Reply:
![image](https://github.com/We-Are-Swissborg/WASB-Front/assets/101713837/dc13cfe8-d2e0-44a0-99ba-d2c21c838bd8)

C'est pour éviter cette erreur